README.md

```diff
@@ -7,8 +7,8 @@ This repo can be installed on the same VM as the main LORIS codebase, or on a di
 
 # System Requirements
  * Perl
- * MINC toolkit (step 2 below)
- * DICOM toolkit (step 3 below)
+ * Python 3 with pip3 and virtualenv (step 2 below)
```

Comment:
Note: I removed the DICOM toolkit from the requirements since it is automatically installed by the install script. Can put it back if anyone has a strong opinion about this.

Reply:
DICOM toolkit was explicitly stated there because for CentOS it's not automatically installed by the install script; if I recall that works only on Ubuntu.   We could likely clarify that + the below sentence.

Reply:
@christinerogers Thanks for the clarification! I put it back to the README. Might be good indeed to clarify this in a separate PR.